odoo_module_migrate/migration_scripts/text_replaces/migrate_120_130/digits_precision.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
This change should not be part of commit "Fix migration script: allow for having several files defining rules f…"